core/src/test/kotlin/org/rewedigital/katana/InjectionTests.kt

```diff
@@ -266,5 +266,49 @@ object InjectionTests : Spek(
 
                 component.custom<String>(name = "TEST", arg = "World") shouldEqual "Hello World"
             }
+
+            it("inject*OrNull functions should return null for non-declared bindings") {
```

Comment:
is this `*` supposed to be there?

Reply:
Yes, I mean to test `injectOrNull` and `injectNowOrNull` in this test. Therefore the `*`.

Reply:
got it 👍 although i would separate them then